spec/factories/member.rb

```diff
@@ -1,6 +1,6 @@
 FactoryGirl.define do
   factory :member do
     sequence(:identifier) { |n| "123coolIdInHerexxxx#{n}" }
-    association :user, factory: :user, username: 'UserMember'
+    association :user, :as_player
```

Comment:
`:member` factory with an `:user` association `:as_player`?

I don't understand it but it looks very strange ;-)